### ISSUE TITLE
Implement adaptive Learning Rate

### ISSUE THREAD

**Issue by HaukeTh** [blumenstiel/TDT4265-project#14]

**Comment by blumenstiel** [comment]
https://www.kaggle.com/isbhargav/guide-to-pytorch-learning-rate-scheduling
